regimes/mx/identities.go

```diff
@@ -0,0 +1,336 @@
+package mx
+
+import (
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+// Mexican CFDI identity keys required by the SAT (tax authority in Mexico) in all
+// invoices and cannot be determined automatically.
+const (
+	IdentityKeyCFDIFiscalRegime = "mx-cfdi-fiscal-regime"
+	IdentityKeyCFDIUse          = "mx-cfdi-use"
+	IdentityKeyCFDIProdServ     = "mx-cfdi-prod-serv" // name from XML field: ClaveProdServ
```

Comment:
I've gone with the `mx-cfdi-` prefix in the end. This makes it super clear what the context is, and namespaces the keys according to the regime.